AutoGraph/AutoGraph.swift

```diff
@@ -86,6 +97,40 @@ open class AutoGraph {
         self.send(requestIncludingJSON, completion: completion)
     }
     
+    open func subscribe<R: Request>(_ request: R, operationName: String, completion: @escaping RequestCompletion<R.SerializedObject>) {
+        guard let webSocketClient = self.webSocketClient else {
+            completion(.failure(AutoGraphError.subscribeWithMissingWebSocketClient))
+            return
+        }
+        
+        webSocketClient.subscribe(request, operationName: operationName, completion: { (result) in
+            switch result {
+            case let .success(data):
+                do {
+                    let serializedObject = try JSONDecoder().decode(R.SerializedObject.self, from: data)
+                    completion(.success(serializedObject))
```

Comment:
serialization should happen on a background queue

Reply:
actually, this happens in background, it should return to the main queue when it's done